### PR TITLE
Add timestamp

### LINKED ISSUE #23: /jobs/{job_id}/ready endpoint should return the timestamp of the time when job finished
This could be easily implemented by

```
finished_at = datetime.fromtimestamp(calendar.timegm(minio.stat_object(request_id, 'done').last_modified))
```

### PR DESCRIPTION
**Pull request by gdynusa**
closes #23

### CODE REVIEW

File: tests/api/test_get_status.py [hunk 1]
```diff
@@ -10,26 +15,48 @@ def test_get_status(client):
...
+    response = client.get('/jobs/uuid-ready/wait_ready')
     assert response.status_code == 200
     assert response.json == {'ready': True}
 
-    response = client.get('/jobs/uuid-5/ready')
+    response = client.get('/jobs/uuid-ready/ready')
     assert response.status_code == 200
-    assert response.json == {'ready': True}
+    assert response.json['ready'] is True
+    assert response.json['finished_at'][:17] == timestamp.isoformat()[:17]
```
Comment (Teyras) [inline tests/api/test_get_status.py]: Why the slice?

File: shepherd/api/views.py [hunk 2]
```diff
@@ -65,7 +68,13 @@ def is_ready(job_id: str):
         :param job_id: An identifier of the queried job
         """
         check_job_exists(minio, job_id)
-        return JobStatusResponse({'ready': shepherd.is_job_done(job_id)})
+        ready = shepherd.is_job_done(job_id)
+        if ready:
+            timestamp = datetime.fromtimestamp(calendar.timegm(minio.stat_object(job_id, 'done').last_modified))
```
Comment (Teyras) [inline shepherd/api/views.py]: Could you break this down into ~2 statements? The nested calls are kinda hard to read.

File: tests/api/conftest.py [hunk 3]
```diff
@@ -17,9 +17,16 @@ def status_gen():
                     "version": "latest"
                 }
             })
+
+    def ready(*args):
+        if args[0] == 'uuid-ready':
```
Comment (Teyras) [inline tests/api/conftest.py]:
```suggestion
        return args[0] == 'uuid-ready'
```

File: tests/api/test_get_status.py (hunk 1 above)
Comment (gdynusa) [reply tests/api/test_get_status.py]: Cause the timestamps can be only approximately same, so I sliced off seconds and fractions of seconds

Comment (Teyras) [reply tests/api/test_get_status.py]: That's... not pretty. Please, at least calculate the difference and assert it is less than a second or something.

Comment (petrbel) [reply tests/api/test_get_status.py]: @gdynusa Btw what would happen if we run the tests around midnight (which circleci does everyday...)

**Pull request merged**